packages/core/src/routes/custom-phrase.ts

```diff
@@ -42,4 +46,22 @@ export default function phraseRoutes<T extends AuthedRouter>(router: T) {
       return next();
     }
   );
+
+  router.delete(
+    '/custom-phrases/:languageKey',
+    koaGuard({
+      params: CustomPhrases.createGuard.pick({ languageKey: true }),
+    }),
+    async (ctx, next) => {
+      const {
+        params: { languageKey },
+      } = ctx.guard;
+
+      await findCustomPhraseByLanguageKey(languageKey);
```

Comment:
If we have the deletion error guard, do we still need to check the existence ahead?

Reply:
Good call. (Add an issue LOG-4166 to improve other DELETE routes.)